src/invent.c

```diff
@@ -2686,6 +2695,14 @@ long *out_cnt;
                  xtra_choice, MENU_ITEMFLAGS_NONE);
         gotsomething = TRUE;
     }
+
+    /* Show weight total and item limit. */
+    char invheading[QBUFSZ];
+    int wcap = weight_cap();
+    Sprintf(invheading, "Inventory: %d/%d weight (%d/52 slots)",
+            inv_weight() + wcap, wcap, inv_cnt(TRUE));
```

Comment:
It looks like including gold here will make this display `(53/52 slots)` for a full inventory. Since `$` is a dedicated inventory slot, is it better to just omit it (rather than increase to 53 slots)?

Reply:
I agree, this would be better off with inv_cnt(FALSE) since gold doesn't occupy a letter slot.